src/index.html

```diff
@@ -89,7 +91,7 @@
   </script> -->
   <!-- Including Ck-edior script-->
   <script src="https://cdn.ckeditor.com/4.6.1/full/ckeditor.js"></script>
-  
+  <script src="https://checkout.razorpay.com/v1/checkout.js"></script>
```

Comment:
Is it possible to load the staging scripts on staging only?

Reply:
@gpalsingh We can use this in **_main.ts_**
`if (!environment.production) {
     document.write('<script src="https://checkout.razorpay.com/v1/checkout.js"></script>');
} `

Reply:
Done